enderio-decoration/src/main/java/com/enderio/decoration/common/network/ServerToClientLightUpdateHandler.java

```diff
@@ -0,0 +1,14 @@
+package com.enderio.decoration.common.network;
+
+import java.util.function.Supplier;
+
+import net.minecraft.client.Minecraft;
+import net.minecraftforge.network.NetworkEvent.Context;
+
+public class ServerToClientLightUpdateHandler {
+
+    public static void handlePacket(ServerToClientLightUpdate message, Supplier<Context> ctx) {
+        Minecraft.getInstance().level.setBlock(message.pos, message.state, 3);
```

Comment:
Could this be done without the need of sending the state over the network and instead querying for the state already present? Might save on traffic.

Reply:
We will need a packet either way, since the state is set from the server. I currently only send this packet to the server, since the flag is 1 on the server. But that needs to be double checked.

Reply:
Yeah, I mean rather than sending the state as part of the message could we not just save the position and find the state based on said position, it'd make the packet smaller.

Reply:
But that means the state is already known on the client, which isn't the case. The client does not have the block (if i read the flags right).